### ISSUE TITLE
Remove Implicit Attribute Relations and Roles in Graql

### ISSUE THREAD

**Issue by flyingsilverfin** [graknlabs/graql#113]
## Problem to Solve
We want to remove implicit attribute relations and roles, as indicated in https://github.com/graknlabs/grakn/issues/5614. In the language, this means we will no longer include any special identifiers for relations using `@`

## Proposed Solution
* Remove anything to do with implicit attributes, including updating the parser if required

## Work to do
- [x] remove the `via` keyword and its usages
- [x] remove the reserved `@` and other implicit relation/role syntax parsing (https://github.com/graknlabs/graql/pull/127)

**Comment by haikalpribadi** [comment]
Do we need a separate issue for this given that we already have #5614, @flyingsilverfin ?

**Comment by flyingsilverfin** [comment]
Yeah, only need one of them I think! This is about the grammar modificaiton directly

**Comment by haikalpribadi** [comment]
Ah, this is in the Graql repo! sorry I didn't notice. Do we even have anything to change in this repo? 🤔 I'm not sure we do...

**Comment by flyingsilverfin** [comment]
There's some tokens in the syntax that need to go using `@`

**Comment by haikalpribadi** [comment]
Before we can work on this issue, we actually need to first implement: https://github.com/graknlabs/graql/issues/115

**Comment by haikalpribadi** [comment]
What's left to be done to close this issue, @flyingsilverfin ? Should we also cleanup the Grammar file to remove any remnant of the implicit relation data structure?

**Comment by flyingsilverfin** [comment]
Can purge the rest yeah

**Comment by haikalpribadi** [comment]
can you make the rest of the todos clear in the issue description, @flyingsilverfin ?
